### PR TITLE
Luk10114/965 integrated windows auth

### PR DESCRIPTION
**Pull request by LDuncAndroid**
https://devtopia.esri.com/runtime/common-samples/issues/965

### CODE REVIEW

File: kotlin/integrated-windows-authentication/src/main/java/com/esri/arcgisruntime/sample/integratedwindowsauthentication/MainActivity.kt [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+
+                // Search the portal for web maps
+                portal.findItemsAsync(PortalQueryParameters("type:(\"web map\" NOT \"web mapping application\")"))
+                    ?.let { portalItemResult ->
+                        portalItemResult.addDoneListener {
+                            try {
+                                portalItemResult.get()?.results?.let { portalItemSetResults ->
+                                    portalItemAdapter.updatePortalItems(portalItemSetResults)
+                                }
+                            } catch (executionException: ExecutionException) {
```
Comment (TADraeseke) [inline kotlin/integrated-windows-authentication/src/main/java/com/esri/arcgisruntime/sample/integratedwindowsauthentication/MainActivity.kt]: Can you catch an ExecutionException || Interrupted exception (Or even just an Exception) and return message from them in a single catch?

Comment (LDuncAndroid) [reply kotlin/integrated-windows-authentication/src/main/java/com/esri/arcgisruntime/sample/integratedwindowsauthentication/MainActivity.kt]: Not in Kotlin I'm afraid: https://youtrack.jetbrains.com/issue/KT-7128

File: kotlin/integrated-windows-authentication/src/main/res/layout/portal_load_state.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content">
+
+    <ProgressBar
+        android:id="@+id/portalLoadStatePorgressBar"
```
[the remark this answers is not in the document]
Comment (LDuncAndroid) [reply kotlin/integrated-windows-authentication/src/main/res/layout/portal_load_state.xml]: Agh, I was making such good Porgress

**Comment by LDuncAndroid** [comment]
@TADraeseke fixed those issues.

**Pull request merged**